presto-hive/src/main/java/io/prestosql/plugin/hive/HivePartitionKey.java

```diff
@@ -55,9 +56,11 @@ public String getValue()
         return value;
     }
 
-    public int getEstimatedSizeInBytes()
+    public long getEstimatedSizeInBytes()
     {
-        return INSTANCE_SIZE + name.length() * Character.BYTES + value.length() * Character.BYTES;
+        return INSTANCE_SIZE +
```

Comment:
This wrapping seems excessive. I understand why you'd do it for a longer list of names, but this class is a key/value pair, no need to be extensible.